dc_utils/templates/dc_base.html

```diff
@@ -1,42 +1,45 @@
 {% extends "dc_base_naked.html" %}
 {% load static %}
 
-{% block body_base %}
-    <body class="ds-width-full">
+{% block body_raw %}
+    <body {% block body_attrs %}class="ds-width-full"{% endblock body_attrs %}>
         <div class="ds-page">
```

Comment:
Testing: I can't get the `ds-page` div and `main` tags to render in EE via dc_utils. 😖